tests/unit/cocoon/test_utilities.py

```diff
@@ -1598,3 +1594,14 @@ def test_identify_cash_items(self, _, cash_flag, remove_cash_items, ground_truth
             ground_truth, list(dataframe["__currency_identifier_for_LUSID"])
         )
         self.assertEqual(mappings_ground_truth, mappings_test)
+
+    def test_strip_whitespace(self):
```

Comment:
Can we please add some test coverage for a dataframe with other types, e.g. dates and numbers.